priv/templates/phx.gen.live/live_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>LiveTest do
+  use <%= inspect context.web_module %>.ConnCase, async: true
```

Comment:
Should this be two separate live views?

Reply:
Btw, `async: true` here won't work for MySQL since they don't have concurrent tests in the sandbox. We either need to remove this or introduce a flag under `:generators` which controls the default value for async here.

```suggestion
  use <%= inspect context.web_module %>.ConnCase
```